### DISCUSSION TITLE
Keeping HTML, CSS and JS in the same file still results in 100% HTML

### DISCUSSION THREAD

**Discussion by chr-bakk** [community/community#40290] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'd like to keep my HTML, CSS and JS in the same fil, but Github say's repository is 100% HTML.
It's not entirely correct XD

**Comment by radar-blip** [comment]
Its ok 😀

**Comment by RubenJ27** [comment]
You could try changing the folder structure a bit.

**Reply by chr-bakk** [reply]
I said I like to keep it all in the same file.

**Reply by RubenJ27** [reply]
Here is an article that explains how to organize your files, the better we organize them the more helpful it will be to work, since what you are trying to do is a bad practice.

https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files

**Reply by chr-bakk** [reply]
Trust me, it's not for me!
I reach classes and ids with search, having multiple files just leaves me with a lot of tedious tasks.
Also, have fun dealing with cached files after updates.
